addons/website/static/src/scss/secondary_variables.scss

```diff
@@ -51,3 +51,7 @@ $o-website-values-palette-number: length($o-website-values-palettes);
 }
 
 $o-theme-navbar-logo-height: o-website-value('logo-height') !default;
+$o-theme-font-number: o-website-value('font-number');
+$o-theme-headings-font-number: o-website-value('headings-font-number');
+$o-theme-buttons-font-number: o-website-value('buttons-font-number');
+$o-theme-navbar-font-number: o-website-value('navbar-font-number');
```

Comment:
Mmh, we don't need those, you can just use the values by calling `o-website-value` when needed.
What you could do however is setting the font itself here.
```scss
$o-theme-font = nth($o-theme-fonts, o-website-value('font-number'));
```
So that you can use `$o-theme-font` in the code without having to type the whole expression with `nth` everywhere.

Also you can remove the `$o-theme-font-number: 1 !default;` in primary variables in any case...